source/ProcessManager.Orchestrations.Tests/Unit/Processes/BRS_045/Shared/MissingMeasurementsLog/V1/EnqueueActorMessagesStep/EnqueueActorMessageActivityTests.cs

```diff
@@ -0,0 +1,247 @@
+            new Mock<ILogger<EnqueueActorMessageActivity_Brs_045_Shared_MissingMeasurementsLog_V1>>().Object,
+            masterDataProviderMock.Object,
+            new MeteringPointReceiversProvider(_timeZone),
+            enqueueActorMessageMock.Object,
+            queryOptionsMock.Object,
+            databricksSqlMock.Object);
+
+        return sut;
+    }
+
+    private Mock<IOptionsSnapshot<DatabricksQueryOptions>> MockQueryOptions()
+    {
+        var queryOptionsMock = new Mock<IOptionsSnapshot<DatabricksQueryOptions>>();
+        queryOptionsMock
+            .Setup(o => o.Get(It.IsAny<string>()))
+            .Returns(new DatabricksQueryOptions { CatalogName = "TestCatalog", DatabaseName = "TestDatabase" });
+        return queryOptionsMock;
+    }
+
+    private Mock<IMeteringPointMasterDataProvider> MockMasterDataProvider()
+    {
+        var masterDataProviderMock = new Mock<IMeteringPointMasterDataProvider>();
+        masterDataProviderMock.Setup(
+                mdp => mdp.GetMasterData(
+                    It.IsAny<string>(),
+                    It.IsAny<Instant>(),
+                    It.IsAny<Instant>()))
+            .Returns(
+                Task.FromResult<IReadOnlyCollection<MeteringPointMasterData>>(
+                [
+                    new(
+                        MeteringPointId: new MeteringPointId("1114567890123456"),
+                        ValidFrom: DateTimeOffset.MinValue,
+                        ValidTo: DateTimeOffset.MaxValue,
+                        CurrentGridAreaCode: new GridAreaCode("404"),
+                        CurrentGridAccessProvider: ActorNumber.Create("1111111111111"),
+                        CurrentNeighborGridAreaOwners: [],
+                        ConnectionState: ConnectionState.Connected,
+                        MeteringPointType: MeteringPointType.ElectricalHeating,
+                        MeteringPointSubType: MeteringPointSubType.Calculated,
+                        Resolution: Resolution.QuarterHourly,
+                        MeasurementUnit: MeasurementUnit.KilowattHour,
+                        ProductId: "???",
+                        ParentMeteringPointId: new MeteringPointId("2224567890123456"),
+                        EnergySupplier: ActorNumber.Create("2222222222222")),
+                ]));
+        return masterDataProviderMock;
+    }
+
+    private IAsyncEnumerable<IDictionary<string, object>> GenerateDatabricksSqlAsyncEnumerable(int transactionsCount)
+    {
+        var start = Instant.FromUtc(2025, 05, 02, 12, 00);
+        var rows = Enumerable.Range(0, transactionsCount)
+            .Select(
+                i =>
+                    new DatabricksSqlStatementApiMissingMeasurementsLogExtensions.MissingMeasurementsLogRowData(
+                        OrchestrationInstanceId: Guid.NewGuid(),
+                        MeteringPointId: GenerateUniqueMeteringPointId(),
+                        Date: start.Plus(Duration.FromMinutes(30 * i))))
```

Comment:
```suggestion
                        Date: start.Plus(Duration.FromMinutes(15 * i))))
```
I would assume that this is either 15 or 60